.gitignore

```diff
@@ -18,3 +18,8 @@ build
 dist
 .idea
 toolchain.tar.gz
+time.json
+sum_BAND_CHG_H2
```

Comment:
when merging PR, I just find these four binary files are detected as "difference", but they are not, therefore I add them here to avoid treat them as diff anymore